OGGExtractor.cpp

```diff
@@ -296,11 +307,11 @@ void OGGExtractor::extractFiles()
       name = name.replace(QRegExp("[^a-zA-Z0-9_- ]"),QString(""));
       if(name.isEmpty())
       {
-        name = tr("found_ogg_%1").arg(i+1);
+        name = getDefaultOutputFilename(i+1,data);
       }
 
       QDir dir(destination);
-      QFile file(dir.absoluteFilePath(tr("%1 - %2.ogg").arg(i + 1, numberLength, 10, QChar('0')).arg(name)));
```

Comment:
If the **numberLength** variable is no longer needed anymore, it should be removed from the beginning of **OGGExtractor::extractFiles()**